### PR TITLE
coverage: add description about release and latest version differencences

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**
Follow up from #3170.

### CODE REVIEW

File: util/coverage/coverage.js [hunk 1]
```diff
@@ -207,6 +207,11 @@ const miscStrings = {
   oopsy: {
     ...zoneGridHeaders.oopsy,
   },
+  // Description about release and latest version differences.
+  description: {
+    en: 'This list contains contents that are not yet included in the latest cactbot release version.<br>So, you may see some contents are not covered by the release version although this list says cactbot covers that content.',
```
Comment (MaikoTan) [inline util/coverage/coverage.js:212]:
```suggestion
    en: 'This list contains contents that are not yet included in the latest cactbot release version.<br>So, you may see some contents are not covered by the release version although this list says cactbot covers that content.',
  ja: 'このリストは最新バージョンに公開されていないコンテンツを含まれています。<br>そのため、リストに含まれているコンテンツは実際に公開されているリリースに存在はしないかもしれぬことをご理解してください。',
  cn: '该列表中可能存在未发布在cactbot最新发行版中的更新内容。<br>也就是说，即便某些副本支持在列表中已显示为支持状态，也可能暂未存在于您下载的版本中。',
```

Comment (quisquous) [reply util/coverage/coverage.js:212]:
```suggestion
    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date.',
```

Proposing a slight rewording here.  I think no need for `<br>` either, as it should wrap normally?

Comment (Jaehyuk-Lee) [reply util/coverage/coverage.js:212]: I thought the content width balance would be better by adding line break.
And... I encountered a problem I didn't noticed. The emoji goes up when the sentence exceeds the width of the page.

K-008

File: util/coverage/coverage.js [hunk 2]
```diff
@@ -207,6 +207,13 @@ const miscStrings = {
   oopsy: {
     ...zoneGridHeaders.oopsy,
   },
+  // Description about release and latest version differences.
+  description: {
+    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date.',
+    ja: 'このリストは最新バージョンに公開されていないコンテンツを含まれています。<br>そのため、リストに含まれているコンテンツは実際に公開されているリリースに存在はしないかもしれぬことをご理解してください。',
+    cn: '该列表中可能存在未发布在cactbot最新发行版中的更新内容。<br>也就是说，即便某些副本支持在列表中已显示为支持状态，也可能暂未存在于您下载的版本中。',
```
Comment (MaikoTan) [inline util/coverage/coverage.js:214]:
```suggestion
    ja: 'このリストは開発中機能や最新リリースバージョンに公開されていないコンテンツを含まれています。リストに含まれているコンテンツは次バージョンに公開される予定があります。また、「quisquous.github.io」からcactbotを使用している場合はこのリストに含まれているコンテンツと一致し、すべてのコンテンツを使えます。',
    cn: '该列表中可能存在正在开发中的功能及未发布在cactbot最新发行版中的更新内容。该列表中显示的更新将会在下一个版本的cactbot发行版中发布。若您使用的是「quisquous.github.io」开头的URL，则更新进度与该列表一致，即该列表中的所有内容均可用。',
```

File: util/coverage/coverage.js [hunk 3]
```diff
@@ -207,6 +207,13 @@ const miscStrings = {
   oopsy: {
     ...zoneGridHeaders.oopsy,
   },
+  // Description about release and latest version differences.
+  description: {
+    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date.',
```
Comment (MaikoTan) [inline util/coverage/coverage.js:212]: @quisquous 
```suggestion
    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the quisquous.github.io version as the url for your overlays, this list will be up to date. Although this is mainly for developers previewing and testing, which means there might be few bugs. See also <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls"here</a> for more detals.',
```

I think if mentioned gh-pages, it should be nice to notice that the gh-pages version is not a stable one.

Comment (quisquous) [reply util/coverage/coverage.js:212]: I don't think it's mainly for developers, for what it's worth.  I think a number of users also use it to get changes quicker than releases happen, but probably not a huge number.

What about removing the added sentences and linking `quisquous.github.io version` to https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls?

File: util/coverage/coverage.js (hunk 1 above)
Comment (quisquous) [reply util/coverage/coverage.js]: I think you could fix both of those (the really long line, and the emoji on the other line) with additions to `coverage.css`.

File: util/coverage/coverage.js (hunk 3 above)
Comment (MaikoTan) [reply util/coverage/coverage.js:212]: @quisquous 
```suggestion
    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">quisquous.github.io version</a> as the url for your overlays, this list will be up to date',
```

You mean like this?

File: util/coverage/coverage.css [hunk 4]
```diff
@@ -25,6 +25,15 @@ body > div {
...
 }
 
+#description-emoji {
+  float: left;
+  margin-right: 5px;
+}
+
+#description-text {
+  display: inline-block;
+}
```
Comment (quisquous) [inline util/coverage/coverage.css:35]:
```suggestion
#description {
  white-space: nowrap;
}

#description-emoji {
  float: left;
  margin-right: 5px;
}

#description-text {
  display: inline-block;
  white-space; normal;
  max-width: 800px;
}
```

What about something like this?

File: util/coverage/coverage.js [hunk 5]
```diff
@@ -362,6 +369,9 @@ document.addEventListener('DOMContentLoaded', () => {
   const languageSelect = document.getElementById('language-select');
   buildLanguageSelect(languageSelect, lang);
 
+  const description = document.getElementById('description');
```
Comment (quisquous) [inline util/coverage/coverage.js:372]:
```suggestion
  const description = document.getElementById('description-text');
```

File: util/coverage/coverage.css (hunk 4 above)
Comment (Jaehyuk-Lee) [reply util/coverage/coverage.css:35]: Ah, max-width! I didn't have that in mind. I was worrying about fixed width of description div. But Others have dynamic width.

Comment (Jaehyuk-Lee) [reply util/coverage/coverage.css:35]: And the white-space attribute, good and thanks. I learned how to handle this div going upside problem.

File: util/coverage/coverage.js [hunk 6]
```diff
@@ -207,6 +207,13 @@ const miscStrings = {
   oopsy: {
     ...zoneGridHeaders.oopsy,
   },
+  // Description about release and latest version differences.
+  description: {
+    en: 'This list may contain content that is in development and is not yet included in the latest cactbot release. Anything that is listed as covered here will be included in the next release of cactbot.  If you are using the <a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">quisquous.github.io version</a> as the url for your overlays, this list will be up to date.',
+    ja: 'このリストは開発中機能や最新リリースバージョンに公開されていないコンテンツを含まれています。リストに含まれているコンテンツは次バージョンに公開される予定があります。また、OverlayPluginのURL欄に「quisquous.github.io」のページのURLを入力している場合はこのリストに含まれているコンテンツと一致し、すべてのコンテンツを使えるになります。',
+    cn: '该列表中可能存在正在开发中的功能及未发布在cactbot最新发行版中的更新内容。该列表中显示的更新将会在下一个版本的cactbot发行版中发布。若您在OverlayPlugin中使用的是「quisquous.github.io」开头的URL，则更新进度与该列表一致，即该列表中的所有内容均可用。',
```
Comment (MaikoTan) [inline util/coverage/coverage.js:214]:
```suggestion
    ja: 'このリストは開発中機能や最新リリースバージョンに公開されていないコンテンツを含まれています。リストに含まれているコンテンツは次バージョンに公開される予定があります。また、OverlayPluginのURL欄に<a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">「quisquous.github.io」のページのURL</a>を入力している場合はこのリストに含まれているコンテンツと一致し、すべてのコンテンツを使えるになります。',
    cn: '该列表中可能存在正在开发中的功能及未发布在cactbot最新发行版中的更新内容。该列表中显示的更新将会在下一个版本的cactbot发行版中发布。若您在OverlayPlugin中使用的是<a href="https://github.com/quisquous/cactbot/blob/main/CONTRIBUTING.md#validating-changes-via-remote-urls">「quisquous.github.io」开头的URL</a>，则更新进度与该列表一致，即该列表中的所有内容均可用。',
```

**Comment by Jaehyuk-Lee** [comment]
hmmm.. the `a` tag is not working.

**Pull request merged**